### ISSUE TITLE
[Suggestion] Rename

### ISSUE THREAD

**Issue by AK2H** [snake-4/Zygisk-Assistant#6]
Zygisk-Assistant ----->  **ZygiskHide**

**Comment by snake-4** [comment]
I agree that the name is not very descriptive but changing the name of the repository now would break the automatic update functionality for all existing users.
